defaults/main.yml

```diff
@@ -41,6 +41,7 @@ postfix_smtpd_banner: '$myhostname ESMTP $mail_name (Ubuntu)'
 postfix_disable_vrfy_command: false
 postfix_message_size_limit: 10240000
 postifx_header_checks_database_type: regexp
+postifx_sender_canonical_maps_database_type: regexp
```

Comment:
This does not look backwards compatible

Reply:
Correct?

Reply:
I can have it default to old config, one sec